testing-plugin/src/test/kotlin/arrow/meta/plugin/testing/ExampleTest.kt

```diff
@@ -2,47 +2,118 @@ package arrow.meta.plugin.testing
 
 import org.junit.Test
 
-class ExampleTest : CompilationTest {
+class ExampleTest {
 
-  //
-  // TODO: waiting for the arrow-annotations release which contains higherkind annotation
-  //    classpaths = listOf(classpathOf("arrow-annotations:x.x.x"))
-  //
+  @Test
+  fun `checks that compiles`() {
+    val compilerTest = CompilerTest(
+      code = {
+        """
+        | fun hello(): String =
+        |   "Hello world!"
+        | 
+        """.source
+      },
+      assert = {
+        listOf(compiles)
+      }
+    )
+    compilerTest.run(interpreter)
```

Comment:
We should hide running from the user since all these tests are expected to produce Unit

Reply:
Done! I recovered the previous `assertThis`. I think it makes sense now:

* There is an assert in the test.
* The user could be able to change the default interpreter.

Thanks for the review @raulraja !